tests/dump_input/fdb/appl_db.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
suggest to cover the "static" mac entry case

Reply:
Changed the test case into static entry since loading via appl_db pretty much is for static mac entry scenario.